jupyterhub/Readme.md

```diff
@@ -0,0 +1,30 @@
+# Jupyter Hub
+
+This is an example of the jupyter extension running with the JupyterHub notebook runner.]
+
+_Note_: Not for production use
+
+## Instructions
+
+Prerequisites
+
+- Helm
+- A Kubernetes cluster to talk to
+- A running Pachyderm cluster
+- Working knowledge of Kubernetes (connecting to services, etc)
+
+Add the Jupyter helm chart repository
+
+```
+helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
+helm repo update
+```
+
+```
+
```

Comment:
Might be possible to remove these blank lines and make the markdown a little more compact.